composer/loggers/logger_hparams.py

```diff
@@ -77,16 +88,18 @@ class WandBLoggerHparams(LoggerCallbackHparams):
     """:class:`~composer.loggers.wandb_logger.WandBLogger` hyperparameters.
 
     Args:
-        project (str, optional): Weights and Biases project name.
-        group (str, optional): Weights and Biases group name.
-        name (str, optional): Weights and Biases run name.
-        entity (str, optional): Weights and Biases entity name.
-        tags (str, optional): Comma-seperated list of tags to add to the run.
-        log_artifacts (bool, optional): Whether to log artifacts. Defaults to False.
-        log_artifacts_every_n_batches (int, optional). How frequently to log artifacts.
-            Default: ``100``. Only applicable if ``log_artifacts`` is True.
-
-        extra_init_params (JSON Dictionary, optional): Extra parameters to pass into :func:`wandb.init`.
+        project (str, optional): See :class:`~composer.loggers.wandb_logger.WandBLogger`.
```

Comment:
The project, group, name, entity, and tags fields do not exist on the WandB class, since the initialize_object method dumps these into the init_kwargs. Can you add an actual docstring here for these fields? (Will probably be a bunch of one-liners).